draft-terbu-sd-jwt-vc.md

```diff
@@ -421,37 +447,78 @@ The following is a non-normative example of a JWT Issuer Metadata including
 
 # Verifiable Presentations
 
-This specification defines the media type `vp+sd-jwt` which describes the VP-SD-JWT with the following components:
+This section defines encoding, validation and processing rules for VP-SD-JWTs.
```

Comment:
the "presentation of VC-SD-JWTs"? VP sounds to formal in my opinion.

Reply:
Created a ticket here https://github.com/vcstuff/draft-terbu-sd-jwt-vc/issues/64